<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarCard</class>
 <widget class="QWidget" name="CarCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>164</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>164</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelCarName">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Diesel Car</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColor">
        <property name="text">
         <string>Color: Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSeatsNumber">
        <property name="text">
         <string>Number of seats: 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTonnage">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Tonnage: 500</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPrice">
        <property name="font">
         <font>
          <pointsize>19</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>$50000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
